src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -1104,23 +1103,11 @@ public ImmutableArray<SymbolDisplayPart> ToDisplayParts(SymbolDisplayFormat form
             CSharpSyntaxNode block,
             CSharpSyntaxNode expression,
             CSharpSyntaxNode syntax,
-            DiagnosticBag diagnostics)
+            DiagnosticBag diagnostics,
+            ErrorCode code)
```

Comment:
Not using a default value just means more code to read.

Replacing the two error codes with one seems reasonable to me.